assets/css/global.css

```diff
@@ -77,3 +77,7 @@ ol > li,
 ul > li {
   @apply ml-4;
 }
+
+.title {
+  font-weight: 500;
```

Comment:
what is the purpose of this class?

Reply:
please avoid adding anything in global unless you know you will be reusing it again somewhere else. 
Usually we write classes on a component level and we see the need to be reused somewhere else we will move it here

Reply:
As I understood, Issa thinks that not all the headings should have the same font weight. So this class would allow us to add a specific font weight to headings that are titles (such as members' names)

Reply:
Thing is, this font weight will be indeed be used repeatedly in the website (i.e. for all members' names)